lib/game_logic.rb

```diff
@@ -0,0 +1,101 @@
+require('rainbow')
+# rubocop:disable Style/GlobalVars,Metrics/MethodLength
+$board = [1, 2, 3, 4, 5, 6, 7, 8, 9]
+
+$winner = false
+
+class Drawer
+  def draw_board
+    board_sign = $board
+    puts("\t#{board_sign[0]} | #{board_sign[1]} | #{board_sign[2]} ")
+    puts "\t---------"
+    puts("\t#{board_sign[3]} | #{board_sign[4]} | #{board_sign[5]} ")
+    puts "\t---------"
+    puts("\t#{board_sign[6]} | #{board_sign[7]} | #{board_sign[8]} ")
+    puts "\n"
```

Comment:
No puts/print and get.chomp outside main.rb file. ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/ruby/OOP/milestone3/only_main_print.png) from Pathwright)

You could return the string and the `puts` it in the `main.rb` file.